server/router/cohort-router.js

```diff
@@ -4,6 +4,6 @@ const cohortRouter = express.Router();
 
 const getCohorts = require('../controllers/cohort/getCohorts');
 
-cohortRouter.get('/cohorts', getCohorts);
+cohortRouter.get(getCohorts);
```

Comment:
why you delete this route ? just keep it 
```suggestion
cohortRouter.get('/cohorts', getCohorts);
```